docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,715 @@
+
+Averaging multiple MSDs can provide a better estimate.
+If all included tracks have the same number of points, then averaging them doesn't change the expected MSD.
+It does however reduce the variance by a factor of :math:`1/M` and brings the distribution closer to a Gaussian.
+This follows from the fact that the MSDs follow a Gamma distribution and the additive property of independent gamma distributions :cite:`michalet2010mean`.
+As a consequence, the procedure used to estimate the appropriate number of points can safely be used as long as the track lengths are roughly the same.
+
+Working with ensemble MSDs leads to much improved estimates of the diffusion constant.
+We will demonstrate this, by comparing the following two procedures:
+
+#. Averaging diffusion constants obtained by calculating them on short tracks and calculating their average (bad).
+#. Performing a single estimate of a diffusion constant for a group of tracks by estimating the ensemble MSD first.
+
+The first of these two methods is not available in Pylake directly, because it is not a recommended procedure.
+As such, it requires a bit more work to make sure that we save it in a similar format as Pylake does::
+
+    # Because our testing function expects a class with a value and std_err attribute
+    # we have to wrap our results in a similar class.
+    class CustomDiffusionEstimate:
+        def __init__(self, value, std_err):
+            self.value = value
+            self.std_err = std_err
+
+    def bad_ols(tracks):
+        """Just "average" the results for the tracks (NOT a good method)"""
+        diffusion_estimates = tracks.estimate_diffusion("ols")
+        avg_diff_est = np.mean([d.value for d in diffusion_estimates])
+        std_diff_est = np.std([d.value for d in diffusion_estimates]) / np.sqrt(len(diffusion_estimates))
+        return CustomDiffusionEstimate(avg_diff_est, std_diff_est)
+
+    def simulate_tracks(snr):
+        """Function used to simulate tracks"""
+        params = snr_to_diffusion_parameters(snr)
+        tracks = [
+            lk.simulation.simulate_diffusive_tracks(**params, steps=8, num_tracks=10)
+            for _ in range(100)
+        ]
+        return tracks, params["diffusion_constant"]
+
+    methods = {
+        "bad_ols": lambda list_of_tracks: [bad_ols(t) for t in list_of_tracks],
+        "ols_ens": lambda list_of_tracks: [t.ensemble_diffusion("ols") for t in list_of_tracks],
+    }
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+    for method, estimates in results.items():
+        plot_accuracy(snrs, estimates, variances[method], label=method);
+
+    plt.ylim([-5, 6])
+
+.. image:: figures/ols_bias.png
+
+We can see that while both result in increased precision, one is not accurate at all for low SNRs (the mean deviates a lot from 1).
```

Comment:
I'd prefer something like
```
We can see that while both result in increased precision, one is not accurate at all (highly biased) for low SNRs (the mean deviates a lot from 1).
```
since highly biased is basically a synonym for inaccurate. Would you be ok with that as well?